src/generic/templates/template.js

```diff
@@ -47,6 +48,23 @@ class Template extends BaseTemplate {
 
     return this
   }
+
+  /**
+   * Add a {@link RBMQuickReply} to the template
+   * @param {RBMQuickReply} quickReply - Required
+   **/
+  addRBMQuickReply(quickReply) {
+    if(!(quickReply instanceof RBMQuickReply)) {
+      throw new Error('addRBMQuickReply argument must be an instance of a QuickReply')
```

Comment:
Shall we change it to below since `QuickReply` is different from `RBMQuickReply`?
```suggestion
      throw new Error('addRBMQuickReply argument must be an instance of a RBMQuickReply')
```